src/components/DoDontExample/do-dont-example.scss

```diff
@@ -119,12 +119,13 @@ p.example__text {
 .example__title {
   @include type-style('body-short-01');
   margin: 0;
-  padding: 0;
+  padding: 0 !important;
```

Comment:
I think if we just change this to `p.example__title` we won't need the important?